golden_response/staging/get_triples/place_type.json

```diff
@@ -3022,7 +3016,7 @@
       "objectTypes": [
         "Class"
       ],
-      "provenanceId": "dc/5n63hr1"
+      "provenanceId": "dc/zgd6823"
```

Comment:
is this okay? @pradh

Reply:
This is not ideal. I think there is some non-determinism when the triples are comparable across provenances for a node.